var/spack/repos/builtin/packages/readline/package.py

```diff
@@ -45,4 +45,6 @@ class Readline(AutotoolsPackage):
     patch('readline-6.3-upstream_fixes-1.patch', when='@6.3')
 
     def build(self, spec, prefix):
-        make('SHLIB_LIBS=-lncursesw')
+        options = ['SHLIB_LIBS=-L%s -lncursesw', spec['ncurses'].prefix.lib]
```

Comment:
Is this getting substituted? I mean, shouldn't the comma `,` be a percent `%` in the list above?

Just out of curiosity, can you also try to see if:
```python
options = ['SHLIB_LIBS={0}'.format(spec['ncurses'].libs.ld_flags)]
```
is working?